### PR TITLE
Update Stytch example app and add mention to Pages Router

### PR DESCRIPTION
**Pull request by chris-stytch**
# Description

This PR updates the Stytch example and adds a link to it in the Pages Router auth documentation external providers list.

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: f022d7f26a264f353898b3b41446672ef57a67dd

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: f022d7f26a264f353898b3b41446672ef57a67dd

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: f022d7f26a264f353898b3b41446672ef57a67dd

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: examples/auth-with-stytch/src/pages/_app.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import { createStytchUIClient } from '@stytch/nextjs/ui'
+
+// We initialize the Stytch client using our project's public token which can be found in the Stytch dashboard
+const stytch = createStytchUIClient(process.env.NEXT_PUBLIC_STYTCH_PUBLIC_TOKEN)
+
+export default function App({ Component, pageProps }) {
+  return (
+    <>
+      <Head>
+        <title>Stytch Next.js Example</title>
```
Comment (leerob) [inline examples/auth-with-stytch/src/pages/_app.js:13]: Could we move this to the App Router?

File: examples/auth-with-stytch/package.json [hunk 2]
```diff
@@ -1,20 +1,22 @@
 {
+  "name": "stytch-nextjs-example",
+  "version": "0.1.0",
```
Comment (leerob) [inline examples/auth-with-stytch/package.json:3]:
```suggestion
```

File: examples/auth-with-stytch/package.json [hunk 3]
```diff
@@ -1,20 +1,22 @@
...
     "build": "next build",
-    "start": "next start"
+    "start": "next start",
+    "lint": "next lint"
   },
   "dependencies": {
-    "@stytch/stytch-react": "^3.0.3",
-    "next": "12.0.7",
-    "next-iron-session": "^4.2.0",
+    "@next/font": "13.2.3",
```
Comment (leerob) [inline examples/auth-with-stytch/package.json:12]: This is built-in now

File: examples/auth-with-stytch/next.config.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
```
Comment (leerob) [inline examples/auth-with-stytch/next.config.js:3]: This can be removed now

**Review by leerob — CHANGES_REQUESTED** [review]

**Comment by leerob** [comment]
Closing as stale (there are outstanding comments + merge conflicts) but we're open to reviewing a new PR to update this example! 🙏

**Pull request closed without merging**